app/HutchManager/Dockerfile

```
@@ -1,9 +1,11 @@
 ﻿FROM node:lts-alpine as spa_build
+COPY pnpm-lock.yaml .
+COPY pnpm-workspace.yaml .
+COPY package.json .
+COPY ./app/manager-frontend /manager-frontend
 RUN npm install -g pnpm
-WORKDIR /manager-frontend
-COPY ../manager-frontend .
 RUN pnpm install --frozen-lockfile
-RUN pnpm run build
+RUN pnpm --if-present run build
```

should filter to the package we want to build, otherwise it will build everything that has a build script.

```suggestion
RUN pnpm --filter manager-frontend build
```